clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -33,7 +34,24 @@ def describe_clusters():
     if sites != "" and etcd_version != "2.2.5":
         local_site_info = " in the local site (" + local_site + ")"
 
-    print "This script prints out the status of the Chronos, Memcached and Cassandra clusters{}.\n".format(local_site_info)
+    print "This script prints the status of the Chronos, Memcached and Cassandra clusters{}.".format(local_site_info)
+
+    plugin_dir = '/usr/share/clearwater/clearwater-cluster-manager/plugins'
+    if os.path.isdir(plugin_dir):
+        plugins = [plugin_name[:-10] for plugin_name in os.listdir(plugin_dir) if
+                plugin_name.endswith('_plugin.py')]
+        plugins.remove('memcached_remote')
```

Comment:
Can you add a comment saying why it's OK to exclude memcached_remote (e.g. Ignoring memcached_remote.py as this is now deprecated).